config/routes.rb

```diff
@@ -2,5 +2,7 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  # root 'users#index'
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] Please kindly nest the `post` resources routes within the `users` resource routes as expected in accordance to the project requirement, this will enable you to retrieve the posts of a specific user, and view a specific post belonging to a user.
```suggestion
   resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

![milert](https://github.com/karanJ2212/RailsBlog/assets/103386183/5943c260-41a4-4abb-8049-7ba78bf9be3b)